File: ContactPicker/src/main/java/com/onegravity/contactpicker/custom/textview/TxtMedium.java
package com.onegravity.contactpicker.custom.textview;

import android.content.Context;
import android.graphics.Typeface;
import android.support.v7.widget.AppCompatTextView;
import android.util.AttributeSet;

/**
 * Created by ToCHe on 11/3/2018 AD.
 */

class TxtMedium extends AppCompatTextView {

    public TxtMedium(Context context) {
        super(context);
        applyFont(context);
    }

    public TxtMedium(Context context, AttributeSet attrs) {
        super(context, attrs);
        applyFont(context);
    }

    public TxtMedium(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        applyFont(context);
    }

    private void applyFont(Context context) {
        Typeface customFont = Typeface.createFromAsset(context.getAssets(), "fonts/set_medium.ttf");
        setTypeface(customFont);
    }
}

File: app/src/main/kotlin/asunder/toche/sccmanagement/custom/textview/TxtMedium.kt
package asunder.toche.sccmanagement.custom.textview

import android.content.Context
import android.graphics.Typeface
import android.support.v7.widget.AppCompatTextView
import android.util.AttributeSet

/**
 *Created by ToCHe on 24/2/2018 AD.
 */
class TxtMedium : AppCompatTextView {

    constructor(context: Context) : super(context){
        applyFont(context)
    }

    constructor(context: Context, attributeSet: AttributeSet) : super(context, attributeSet){
        applyFont(context)
    }

    constructor(context: Context, attributeSet: AttributeSet, defStyleAttr: Int) : super(context, attributeSet, defStyleAttr){
        applyFont(context)
    }

    private fun applyFont(context: Context) {
        val customFont = Typeface.createFromAsset(context.assets, "fonts/set_medium.ttf")
        typeface = customFont
    }
}